gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
```

Comment:
It is better to leave out the generated Gradle tools unless the Gradle version has changed.